src/server/game/Entities/Player/Player.h

```diff
@@ -2631,6 +2631,15 @@ class Player : public Unit, public GridObject<Player>
 
         static std::unordered_map<int, bgZoneRef> bgZoneIdToFillWorldStates; // zoneId -> FillInitialWorldStates
 
+        // Premium characters and accounts
+        int GetCharacterPremiumLevel(uint32 guid);
+        bool CreateCharacterPremiumLevel(uint32 guid, int premiumLevel);
+        bool DeleteCharacterPremiumLevel(uint32 guid);
+        int GetAccountPremiumLevel(uint32 guid);
+        bool CreateAccountPremiumLevel(uint32 guid, int premiumLevel);
+        bool DeleteAccountPremiumLevel(uint32 guid);
+        int GetCharacterAccount(uint32 guid);
+
```

Comment:
https://isocpp.org/wiki/faq/const-correctness#overview-const

Some thoughts about the keyword `const` - Your choice.